packages/@okta/i18n/src/properties/login.properties

```diff
@@ -4,6 +4,7 @@
 #
 ########################################
 signout = Sign Out
+signin = Sign in
```

Comment:
looks like shall say `Sign In`

Reply:
Not sure. I know it is not consistent with 'Sign Out', but it is not consistent everywhere. Plus the mock says 'Sign in'.

Reply:
It's kind of consistent.
- `Sign in xxxx`
- or `Sign In` (when just `sign in`, first char of both word are upper case)

btw, I dont see any links to mocks but can you doule check with UX?

Reply:
yeah, we should bring it up to UX that we have `Sign In` and `Sign in` and we should try and be consistent

Reply:
The mock is in figma. I just checked, it is now updated to `Sign In`.